### ISSUE TITLE
Datagrid touching first row hits header instead

### ISSUE THREAD

**Issue by anton-kurilov** [infor-design/enterprise-ng#1366]
**Describe the bug**
Prerequisites: Chrome, Chrome Mobile Emulator enabled, simulate touch events
When clicking on the first row of a datagrid the the event is actually captured by the header. This is especially problematic for users on a mobile device that are using a datagrid. In our example it is a 1 column datagrid that represents look up data.

**To Reproduce**
1. Go to ['...'](https://main-enterprise.demo.design.infor.com/components/datagrid/example-filter.html) 
2. Click on the first row in the grid
3. See that the event is captured header instead (screenshot below)
![image](https://user-images.githubusercontent.com/14839675/187254783-491d0d3b-a75e-41a3-bafb-03e902ecb028.png)

4. Note: This does not seem to happen in a desktop view with a mouse event instead of a touch event.

**Expected behavior**
Cell captures event just like on desktop, for example:
![image](https://user-images.githubusercontent.com/14839675/187254909-c24bc1ef-cedb-4ea4-beef-e01b611dc06a.png)

**Version**
latest ng-enterprise , and enterprise 

**Screenshots**
Screenshots already attached

**Platform**
 - Device (if applicable) [Mobile]
 - OS Version: [Chrome Browser Mobile Emulator on Windows]
 - Browser Name [Chrome Browser Mobile Emulator]
 - Browser Version [latest]

**Additional context**

**Comment by tjamesallen15** [comment]
Hello, @tmcconechy. I tried to locate the issue in the project through debugging but I can't find any suitable way on fixing this touch issue since the events are firing normal and the css structure is fine. I talked to @ericangeles regarding this matter and came up that we can skip this since this is a really minor issue and has a workaround for it.

**Comment by tmcconechy** [comment]
Yeah i dont know either @tjamesallen15 lets park this. At least it works in the new web components..

**Comment by anton-kurilov** [comment]
@LilBiggs, just fyi to keep an eye on this github issue since we had to code something in our automation to workaround this issue, and eventually might be reported by mobile first users using lookups that have a datagrid.
